src/shared/sleep-config.c

```diff
@@ -526,6 +536,88 @@ int get_total_suspend_interval(Hashmap *last_capacity, usec_t *ret) {
         return 0;
 }
 
+static int read_power_supply_battery_alarm_file(const char *alarm_filepath, int *ret) {
+        _cleanup_free_ char *batterytrippoint = NULL;
+        int battery_alarm, r;
+
+        r = read_one_line_file(alarm_filepath, &batterytrippoint);
+        if (r < 0)
+                return r;
+
+        r = safe_atoi(batterytrippoint, &battery_alarm);
+        if (r < 0)
+                return r;
+
+        *ret = battery_alarm;
+
+        return 0;
+}
+
+bool battery_trip_point_alarm_exists(void) {
```

Comment:
Errors can be returned, so should be int here

Reply:
fixed